erpnext/stock/doctype/serial_and_batch_bundle/serial_and_batch_bundle.py

```
@@ -2161,6 +2153,38 @@ def get_reserved_serial_nos_for_pos(kwargs):
+		if kwargs.get("serial_nos") and entry.serial_no in kwargs.get("serial_nos"):
+			frappe.throw(
+				_(
+					"The Serial No {0} is reserved against the {1} {2} and cannot be used for any other transaction."
+				).format(bold(entry.serial_no), entry.voucher_type, bold(entry.voucher_no)),
+				title=_("Serial No Reserved"),
+			)
+
+	return serial_nos
+
+
+def get_other_doc_reserved_serials(kwargs, reserved_entries, reserved_voucher_details):
```

Ruff (0.13.1)

Unused function argument: `kwargs`

(ARG001)